PWGCF/FemtoDream/FemtoDreamTrackSelection.h

```diff
@@ -160,7 +160,8 @@ class FemtoDreamTrackSelection : public FemtoDreamObjectSelection<float, femtoDr
   static int findSelectionIndex(const std::string_view& obs, std::string_view prefix = "")
   {
     for (int index = 0; index < kNtrackSelection; index++) {
-      std::string_view cmp{static_cast<std::string>(prefix) + static_cast<std::string>(mSelectionNames[index])};
+      std::string comp = static_cast<std::string>(prefix) + static_cast<std::string>(mSelectionNames[index]);
+      std::string_view cmp{comp};
```

Comment:
Why not in a single line?

Reply:
I think there is some memory management issue. For some reason, an one-liner results in cmp having spurious content. We need this explicit std::string comp.

Reply:
I see
Interesting but also frightening!